### PR TITLE
release: 0.8.0

### PR DESCRIPTION
**Pull request by askonev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "build": "ng build --configuration production",
     "build-aot-disable": "ng build --configuration production --aot=false --build-optimizer=false",
     "docker-build": "/bin/bash docker-build/build.sh",
-    "postinstall": "ngcc"
+    "postinstall": "ngcc",
+    "add-repo-tag": "make add-repo-tag"
```
Comment (VitaliySerov) [inline package.json:13]: Do we need that in packages file? Do we have a scenario when we want to run it via JS, not via bash command `make add-repo-tag` directly?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.DEFAULT_GOAL := help
```
Comment (VitaliySerov) [inline makefile:1]: I think usually makerfiles are called with capital letter, `Makefile`

But nice choice :)

**Review by VitaliySerov — APPROVED** [review]
Good but several minor notes

**Comment by askonev** [comment]
> Good but several minor notes

Palladium production has a version already 0.7.1 

And the tag 0.6.5 is installed in github

Can I install 0.7.1 release right away in this PR?

image


image

File: package.json (hunk 1 above)
Comment (askonev) [reply package.json:13]: I thought it was a good idea to have one entry point to control the project

Comment (askonev) [reply package.json:13]: There is no special technical reason to add a launch via npm ng

**Comment by askonev** [comment]
image
While I've been testing with a fool, I've added two tags - I'll delete them

File: package.json (hunk 1 above)
Comment (VitaliySerov) [reply package.json:13]: Maybe in that case in README should be only npm variant, so if we need to change realisaztion (from example moving from Makefile to something else) in all cases we will use NPM command

**Comment by VitaliySerov** [comment]
Maybe to solve all the mess with tags just call this new future version v0.8.0 and forget about all older non-correct tags

Since seems like we didn't have any process back in time and the version numbering can be really messy

**Comment by VitaliySerov** [comment]
One more important thing:

https://github.com/palladium-tms/palladium-view/blob/master/package.json#L3

This line for v0.7.1

I think we need to add to update version instruction info about increasing it too

**Pull request merged**